client/landing/stepper/declarative-flow/internals/components/migration-assistance-modal/index.tsx

```diff
@@ -1,7 +1,14 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { useTranslate } from 'i18n-calypso';
 import ConfirmModal from 'calypso/blocks/importer/components/confirm-modal';
 import { setMigrationAssistanceAccepted } from 'calypso/blocks/importer/wordpress/utils';
 import './style.scss';
+
+const EVENT_NAMES = {
+	accepted: 'calypso_migration_assistance_modal_deal_accepted',
+	declined: 'calypso_migration_assistance_modal_no_thanks_or_closed',
```

Comment:
Can we change the name to `...assistance_modal_deal_declined`?
Maybe add another invite as well for ignoring the deal.

Reply:
I have searched in calypso and didn't find anything related, but `deal_declined` seems more readable 👍